homeassistant/components/amcrest/sensor.py

```diff
@@ -68,22 +64,22 @@ def icon(self):
     @property
     def unit_of_measurement(self):
         """Return the units of measurement."""
-        return SENSORS.get(self._sensor_type)[1]
+        return self._unit_of_measurement
 
     def update(self):
         """Get the latest data and updates the state."""
         _LOGGER.debug("Pulling data from %s sensor.", self._name)
 
         if self._sensor_type == 'motion_detector':
-            self._state = self._camera.is_motion_detected
-            self._attrs['Record Mode'] = self._camera.record_mode
+            self._state = self._api.is_motion_detected
+            self._attrs['Record Mode'] = self._api.record_mode
```

Comment:
State attribute keys should be lowercase snakecase.

Reply:
Yes, planned for another PR. In fact, this whole sensor type will be removed, which will itself already be a breaking change. Trying to avoid this PR being a breaking change if at all possible.

Reply:
Aren't we moving services already, ie a breaking change?

Reply:
No, adding services that never existed. (When I originally wrote them almost a year ago as part of my custom component version it was before the great migration and the services were added to `camera/amcrest.py`. The "move" is happening as part of me trying to fold those new features into the standard component and realizing they now probably belonged in `amcrest/__init__.py` instead of `amcrest/camera.py`. Does that make sense?)

Reply:
Right. If there's no breaking changes in this PR, it's ok to wait with those changes to another PR, and then try to group all of them together.